doc/release-notes/rl-2505.section.md

```diff
@@ -4,6 +4,10 @@
 
 <!-- To avoid merge conflicts, consider adding your item at an arbitrary place in the list instead. -->
 
+- `services.rippled` has been removed, as `rippled` was broken and had not been updated since 2022.
+
+- `services.rippleDataApi` has been removed, as `ripple-data-api` was broken and had not been updated since 2022.
+
```

Comment:
These are the [new](https://github.com/NixOS/nixpkgs/pull/371930) packages-specific release notes, so it's not the right place for NixOS modules changes like this, which should go to https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md instead.

@christoph-heiss Could you fix this with a follow-up PR?

Reply:
True, though I'm not sure when Valentin can get to finishing that (not sure why it's it was marked as a draft tbh, it seemed mostly okay), and I think it's more important to not have NixOS changes in the packages release notes (as in this PR), rather than the other way around (which was the standard until recently).

Alternatively, would you be interested to finish Valentin's PR? :D